bot/exts/info/tags.py

```diff
@@ -160,7 +159,8 @@ async def _send_matching_tags(self, ctx: Context, keywords: str, matching_tags:
     @group(name='tags', aliases=('tag', 't'), invoke_without_command=True)
     async def tags_group(self, ctx: Context, *, tag_name: TagNameConverter = None) -> None:
         """Show all known tags, a single tag, or run a subcommand."""
-        await self.get_command(ctx, tag_name=tag_name)
+        # Use invoke here for testability purposes
+        await ctx.invoke(self.get_command, tag_name=tag_name)
```

Comment:
I'm not sure what the difference between these two is, and why one is easier to test than the other (based on the commit message that adds this diff). Would you be able to expand on that please?